source/catalog.html

```diff
@@ -0,0 +1,258 @@
+              <dd class="catalog-list__description-text">500 г</dd>
+              <dt class="catalog-list__description-name">Вкус</dt>
+              <dd class="catalog-list__description-text">Гречка</dd>
+              <dt class="catalog-list__description-name">Цена</dt>
+              <dd class="catalog-list__description-text">400 Р.</dd>
+            </dl>
+            <button class="catalog-list__button button" type="button">
+              заказать
+            </button>
+          </li>
+
+          <li class="catalog-list__item">
+            <h2 class="catalog-list__title">Cat Energy slim 1000 г</h2>
+            <img class="catalog-list__image" src="#" alt="Фотография товара." />
+            <dl class="catalog-list__description-list">
+              <dt class="catalog-list__description-name">Масса</dt>
+              <dd class="catalog-list__description-text">1000 г</dd>
+              <dt class="catalog-list__description-name">Вкус</dt>
+              <dd class="catalog-list__description-text">Гречка</dd>
+              <dt class="catalog-list__description-name">Цена</dt>
+              <dd class="catalog-list__description-text">700 Р.</dd>
+            </dl>
+            <button class="catalog-list__button button" type="button">
+              заказать
+            </button>
+          </li>
+
+          <li class="catalog-list__item">
+            <h2 class="catalog-list__title">Cat Energy slim 500 г</h2>
+            <img class="catalog-list__image" src="#" alt="Фотография товара." />
+            <dl class="catalog-list__description-list">
+              <dt class="catalog-list__description-name">Масса</dt>
+              <dd class="catalog-list__description-text">500 г</dd>
+              <dt class="catalog-list__description-name">Вкус</dt>
+              <dd class="catalog-list__description-text">Рис</dd>
+              <dt class="catalog-list__description-name">Цена</dt>
+              <dd class="catalog-list__description-text">500 Р.</dd>
+            </dl>
+            <button class="catalog-list__button button" type="button">
+              заказать
+            </button>
+          </li>
+        </ul>
+        <div class="catalog__show-all">
+          <img class="catalog__image" src="#" alt="#" />
+          <p class="catalog__subtitle"><b>Показать еще 100500 товаров</b></p>
+          <p class="catalog__text">На самом деле вкусов гораздо больше!</p>
+          <button class="catalog__button button">показать все</button>
+        </div>
+      </section>
+      <section class="additional">
+        <h2 class="additional__title">Дополнительные товары</h2>
+        <ul class="additional-list">
+          <li class="additional-list__item">
+            <h3 class="additional-list__title">Сахарозаменитель</h3>
+            <div class="about-list__description">
+              <p class="additional-list__product-details">
+                <span class="additional-list__value">1 упаковка (100 г)</span>
```

Comment:
```suggestion
                <span class="additional-list__details-value">1 упаковка (100 г)</span>
```
может так стоит писать? или нет? надо посмотреть практики